transcendence-app/src/chat/chat.controller.ts

```diff
@@ -98,6 +99,24 @@ export class ChatController {
     return chatRooms;
   }
 
+  @Get('room/:chatRoomId/members')
+  @ApiOkResponse({
+    description: `Lists all chat members for a given room)`,
+    type: [ChatmemberAsUserResponseDto],
+  })
+  @ApiBadRequestResponse({ description: 'Bad Request' })
+  @ApiServiceUnavailableResponse({ description: 'Service unavailable' })
+  async retrieveChatRoomMembers(
+    @Param('chatRoomId', ParseUUIDPipe) chatRoomId: string,
+  ): Promise<ChatmemberAsUserResponseDto[]> {
+    const chatRoomsMembers =
+      await this.chatMemberService.retrieveChatRoomMembers(chatRoomId);
```

Comment:
maybe we could through a Forbidden response if the auth user is not a member of the chatroom.

Reply:
done